lib/config/folderconfiguration.go

```diff
@@ -164,6 +166,11 @@ func (f *FolderConfiguration) prepare() {
 		f.RescanIntervalS = 0
 	}
 
+	if f.NotifyDelayS <= 0 {
+		f.FsNotifications = false
```

Comment:
Not sure you need this either, unless you want 0 value on delay to disable notifications.